geocoder_tester/base.py

```diff
@@ -134,23 +194,21 @@ def compare_values(get, expected):
 def assert_search(query, expected, limit=1,
                   comment=None, lang=None, center=None,
                   max_matches=None):
-    params = {"q": query, "limit": limit}
+    query_limit = max(CONFIG['CHECK_DUPLICATES'] or 0, limit)
```

Comment:
This breaks when limit is a string read from a .csv :scream: 
We should convert `limit` to an int first.

Reply:
:sweat: